cli/runner/cmd/run.go

```diff
@@ -10,12 +10,21 @@ import (
 	"github.com/spf13/cobra"
 )
 
+/*
+	Example Usage:
+	runner run source.py
+	runner run -l source.py # HAS CLI CHECK FOR LANGUAGE
```

Comment:
Awesome, thanks for including this! You can actually write this into the `&cobra.Command{}` directly so the user can see this too when they run with `--help` flag.

```go
&cobra.Command{
	Use:   "run",
	Short: "runs the supplied code loaded in from a file.",
    Long: `
Example Usage:
	runner run source.py
	runner run -l source.py # HAS CLI CHECK FOR LANGUAGE
`
    ...
}
```